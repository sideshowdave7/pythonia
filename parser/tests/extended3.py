"bob"[3:2:1]
"abcdefg"[::-1]
"string"[:] 
"moar"[:2:]
"less"[2:]
"why"[2::]
[1, 2, 3][0:1]

