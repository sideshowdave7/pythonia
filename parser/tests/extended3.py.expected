(program
 (expr (indexed "bob" (subscript 3 2 1)))
 (expr (indexed "abcdefg" (subscript #f #f ("-" 1))))
 (expr (indexed "string" (subscript #f #f #f)))
 (expr (indexed "moar" (subscript #f 2 #f)))
 (expr (indexed "less" (subscript 2 #f #f)))
 (expr (indexed "why" (subscript 2 #f #f)))
 (expr (indexed (list 1 2 3) (subscript 0 1 #f))))

